src/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj

```diff
@@ -97,6 +97,7 @@
       <CompileAs>CompileAsCpp</CompileAs>
       <IntrinsicFunctions>true</IntrinsicFunctions>
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
+      <LanguageStandard>stdcpp17</LanguageStandard>

```

Comment:
is there any size change for the output assemblies? have you tested it on Win7 box?

Reply:
Size is the same. I don't see why I would need to test this on a win7 box, but sure.

Reply:
Looks fine on win7.